fastify.js

```diff
@@ -424,13 +424,23 @@ function fastify (options) {
 
   function setValidatorCompiler (validatorCompiler) {
     throwIfAlreadyStarted('Cannot call "setValidatorCompiler" when fastify instance is already started!')
+
+    if (!this[kValidatorCompiler]) {
+      this.ready(() => {
+        for (const hook of this[kHooks].onCompilerReady) hook.call(this, 'validator', validatorCompiler)
+        this[kChildren].forEach(child => { for (const asd of child[kHooks].onCompilerReady) asd.call(child, 'validator') })
+      })
```

Comment:
Can you use the `'preReady'` event in avvio?

Reply:
It works.. so the `preReady` event is fired only for registered function and not `_bound after`

Gotcha!